Cabal/src/Distribution/PackageDescription/Check/Target.hs

```diff
@@ -61,6 +61,8 @@ checkLibrary
           _libVisibility_
           libBuildInfo_
         ) = do
+    mapM_ checkModuleName (explicitLibModules lib)
```

Comment:
Curious, what does `explicitLibModules` pick? Not internal libraries? If so, does it make sense?

Reply:
```
explicitLibModules :: Library -> [ModuleName]
explicitLibModules lib =
  exposedModules lib
    ++ otherModules (libBuildInfo lib)
    ++ signatures lib
```

To my understanding, an internal library is a separate component, so that will work out fine?

Reply:
Ah of course an internal library *is* still a library, very good.